fvtest/gctest/GCConfigTest.cpp

```diff
@@ -52,11 +52,40 @@ void
 GCConfigTest::SetUp()
 {
 	OMRPORT_ACCESS_FROM_OMRPORT(gcTestEnv->portLib);
+	const char *param = GetParam();
```

Comment:
Is this fixing a defect or something? Or just an artifact of dev?

Reply:
This is one of the test changes I will remove